README.md

```diff
@@ -1,58 +1,62 @@
+<h1 align="center">Module Showcase Website</h1>
+
+<h5 align="center">Keeping track of Terasology's modules since 2019.</h5>
+
+[![status](https://img.shields.io/badge/status-pre--alpha-red.svg)](https://github.com/Terasology/terasology.github.io)
+[![License](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://opensource.org/licenses/MIT)
+[![GitHub issues](https://img.shields.io/github/issues/Terasology/terasology.github.io.svg)](hhttps://github.com/Terasology/terasology.github.io/issues/)
+
+---
+
 <p align="center">
-  <h3 align="center">Module Showcase Website</h3>
-  <p align="center">
-    Keeping track of Terasology's modules since 2019.
-  </p>
+  <a href="#setup">About</a>&nbsp;&nbsp;
+  <a href="#help">Setup</a>&nbsp;&nbsp;
+  <a href="#testing">Testing</a>&nbsp;&nbsp;
+  <a href="#contributing">Contributing</a>
 </p>
 
-## Table of Contents
+---
 
-* [About the Project](#about-the-project)
-  * [Built With](#built-with)
-* [Getting Started](#getting-started)
-  * [Prerequisites](#prerequisites)
-  * [Setup](#setup)
+<h2 id="about">About The Project</h2>
 
-## About The Project
 Terasology’s basic engine can be extended by a huge amount (201 right now!) of modules. Keeping track of them is not the easiest task. This is why this gatsby framework provides an automated generator for a website, listing all of them. This showcase website will increase discoverability by allowing to filter and search for modules by keywords and categories.
 
-### Built With
+<h2 id="built-with">Built With</h2>
+
 The following generator is built using the following software, you'll need them installed in your workspace to run properly. 
 * [Node](https://nodejs.org/en/)
 * [Gatsby](https://www.gatsbyjs.org/)
```

Comment:
Minor: Both Node and Yarn have extra (if minimal) steps about installing them, yet Gatsby doesn't. Consistency good :-)

For me on my Mac it was just `npm install -g gatsby-cli` after I had installed Node and Yarn